.github/workflows/update-images.yaml

```diff
@@ -89,9 +89,9 @@ jobs:
         credentials_json: '${{ secrets.GKE_KEY }}'
 
     - name: 'Set up Cloud SDK'
-      uses: 'google-github-actions/setup-gcloud@v0'
+      uses: 'google-github-actions/setup-gcloud@v2'
       with:
-        version: '319.0.0'
+        version: 'latest'
```

Comment:
I think we should pin this, otherwise it'll break unexpectedly - gcloud follows latest, python/ubuntu is stuck on old